components/BurgerMenu/CollapseMenu.tsx

```diff
@@ -0,0 +1,121 @@
+import clsx from "clsx";
+import Link from "next/link";
+import { useState } from "react";
+
+import { getLinkPath } from "@/lib/menus";
+import { translate } from "@/lib/translations";
+import { MenuItemWithChildrenFragment } from "@/saleor/api";
+
+import { useRegions } from "../RegionsProvider";
+import styles from "./BurgerMenu.module.css";
+import ChebronDown from "./chevronDownIcon.svg"; // in the final version it should be imported from ui-kit package
+
+interface CollapseMenuProps {
+  data: MenuItemWithChildrenFragment;
+}
+
+function SubCollapseMenu({ data }: CollapseMenuProps) {
+  const [open, setOpen] = useState(false);
+  const { currentChannel, currentLocale } = useRegions();
+
+  const anchor = data.url ? (
```

Comment:
Since this is practically the same (except classnames) in both of these component, it should probably be a component as well, e.g `MenuAnchor` or whatever. Then we could just 
```
 <div className="mt-4">
      {!data?.children?.length ? (
          <MenuAnchor
```